portal/views/portal.py

```diff
@@ -144,6 +144,18 @@ def exercise_and_diet():
 def lived_experience():
     return render_template('gil/lived-experience.html', user=current_user())
 
+@portal.route('/stories/david-andrew-story')
+def david_andrew_story():
```

Comment:
These could probably all be served by a single view function, which should reduce some code duplication and should simplify the creation of language-specific versions of each GIL page (eg `gil/lived-experience_en_AU.html`).

There's a [good example here](http://flask.pocoo.org/snippets/126/), but let me know if you need any help :)